VultisigApp/VultisigApp.xcodeproj/project.pbxproj

```diff
@@ -2343,15 +2347,15 @@
 				IPHONEOS_DEPLOYMENT_TARGET = 17.0;
 				LD_RUNPATH_SEARCH_PATHS = "@executable_path/Frameworks";
 				"LD_RUNPATH_SEARCH_PATHS[sdk=macosx*]" = "@executable_path/../Frameworks";
-				MACOSX_DEPLOYMENT_TARGET = 14.5;
+				MACOSX_DEPLOYMENT_TARGET = 14.0;
```

Comment:
Can you please set the MACOS_DEPlOYMENT_TARGET = 14.5? Also keep SUPPORTS_MAC_DESIGNED_FOR_IPHONE_IPAD = YES